tracer/src/Datadog.Trace/ServiceFabric/PropagationContext.cs

```diff
@@ -1,25 +0,0 @@
-// <copyright file="PropagationContext.cs" company="Datadog">
-// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
-// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
-// </copyright>
-
-#nullable enable
-
-namespace Datadog.Trace.ServiceFabric
-{
-    internal readonly struct PropagationContext
```

Comment:
Was this unused?

Reply:
Looks like it. I only found it because it has a `SamplingPriority` and I couldn't find any uses.